src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -67,7 +67,8 @@ public Measure updateMeasure(
             measureUtil.validateAllMeasureGroupReturnTypes(updateElm(updatingMeasure, accessToken));
         // no errors were encountered so remove the ELM JSON error
         // TODO: remove this when backend validations for CQL/ELM are enhanced
```

Comment:
Is this comment still relevant?

Reply:
I am not sure about this Greg, I haven't worked on it. This change is seen based on my checkstyle